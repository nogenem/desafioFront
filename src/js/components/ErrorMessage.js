const renderErrorMessage = text => `
  <div id="errorAlert" class="alert alert-danger text-center" role="alert">
    <strong>${text}</strong>
  </div>
`;

export default {
  render: renderErrorMessage
};
