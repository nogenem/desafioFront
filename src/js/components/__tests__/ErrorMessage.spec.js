import ErrorMessage from "../ErrorMessage";

it("renders correctly", () => {
  const output = ErrorMessage.render("Some error");
  expect(output).toMatchSnapshot();
});
