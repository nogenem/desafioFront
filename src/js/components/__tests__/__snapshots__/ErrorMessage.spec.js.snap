// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
"
  <div id=\\"errorAlert\\" class=\\"alert alert-danger text-center\\" role=\\"alert\\">
    <strong>Some error</strong>
  </div>
"
`;
